modules/cnv-monitoring-upgrade-status.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/upgrading-container-native-virtualization.adoc
+
+[id="cnv-monitoring-upgrade-status_{context}"]
+= Monitoring upgrade status
+
+The best way to monitor {CNVProductName} upgrade status is to watch the
+ClusterServiceVersion (CSV) `PHASE`.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Run the following command:
++
+----
+$ oc get csv
+----
+
+. Review the output, checking the `PHASE` field. For example:
++
+----
+VERSION  REPLACES                                        PHASE
+2.1.1    kubevirt-hyperconverged-operator.v2.1.0         Installing
+2.1.0                                                    Replacing
+----
+
+. (Optional) Monitor the aggregated status of all {CNVProductName} component
+`Conditions` by running the following command:
++
+----
+$ oc get hco -n openshift-cnv hyperconverged-cluster \
+-o=jsonpath='{range .status.conditions[*]}{.type}{"\t"}{.status}{"\t"}{.message}{"\n"}{end}'
+----
++
+A successful upgrade results in the following output:
++
+----
+ReconcileComplete  True  Reconcile completed successfully
+Available          True  Reconcile completed successfully
+Progressing        False Reconcile completed successfully
+Degraded           False Reconcile completed successfully
+Upgradeable        True  Reconcile completed successfully
+----
+
+[NOTE]
+====
+`PHASE` and `Conditions` both represent the system's best guess based on
```

Comment:
`PHASE` is called out in the first part of the procedure. I think that `Conditions` needs to be added to the sample output for the last command.

Reply:
After getting a little more information, I think this note needs to be removed.

Reply:
>> I don't see either PHASE or Conditions called out.
Phase and Conditions are not things to be called out, it's like a variable displaying the status of the things. Phase title is displayed by `oc get csv` command (value is Installing/Replacing/ Succeeded, ..), and conditions by that complicated yaml parsing command. 
>> After getting a little more information, I think this note needs to be removed.
Bring it back please. Currently, we need to stress, that e.g. even if the Phase reports "Succeeded", it may happen that not all things went/work well, therefore it's "best guess based on... "

Reply:
Opened PR #17095 to address this.